js/reference.js

```diff
@@ -4710,6 +4987,30 @@
         }
         return res;
     };
+    
+    /**
+     * Represents entity filters that can be applied to facet.
+     * 
+     * @param       {ERMrest.tuple} tuple the tuple object
+     * @constructor
+     */
+    function EntityFacetFilter(tuple, col) {
+        
+        ChoiceFacetFilter.superClass.call(this, tuple._data);
+        this.tuple = tuple;
+        this.facetFilterKey = "choices";
+    }
+    module._extends(EntityFacetFilter, FacetFilter);
+    
+    /**
+     * String representation of entity filter. It will be the tuple displayname
+     * 
+     * @return {string}
+     */
+    EntityFacetFilter.prototype.toString = function () {
+        //TODO: should it be unformatted or not?
```

Comment:
This will depend on what the `toString` is going to be used for. If it's for display purposes then yes it should be formatted.